x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_editor/system_prompt/index.test.tsx

```diff
@@ -117,7 +117,8 @@ describe('SystemPrompt', () => {
     });
   });
 
-  describe('when a new prompt is saved', () => {
+  // TODO: To be implemented as part of the global settings tests instead of within the SystemPrompt component
+  describe.skip('when a new prompt is saved', () => {
```

Comment:
this may be unskipped as a refactored test in a follow up PR, per our out of band conversation